generators/gae/index.js

```diff
@@ -649,7 +649,7 @@ module.exports = class extends BaseGenerator {
                 const name = this.gcpCloudSqlInstanceName;
                 // for mysql keep default options, set specific option for pg
                 const dbVersionFlag =
-                    this.prodDatabaseType === 'postgresql' ? ' --database-version="POSTGRES_9_6" --tier="db-g1-small"' : '';
+                    this.prodDatabaseType === 'postgresql' ? ' --database-version="POSTGRES_10" --tier="db-g1-small"' : '';
```

Comment:
Are we adding support for PostgreSQL 10 specifically? Not anything higher than that? Just wondering since otherwise if we do support the current version 12, we can drop this `--database-version` flag completely.

Reply:
@SudharakaP I was just maintaining the default value, the version needs now to be >= 10